src/http/http_message_parser.h

```diff
@@ -38,6 +38,9 @@ namespace dsn {
 
 DEFINE_CUSTOMIZED_ID(network_header_format, NET_HDR_HTTP)
 
+// Number of blobs that a message_ex contains.
+#define HTTP_MSG_BUFFERS_NUM 4
```

Comment:
I think `const uint8_t` is better here

Reply:
Using constexpr or macro is basically the same, but using `const` is not a good practice, because the var is loaded in runtime.